plugins/show.py

```diff
@@ -419,17 +420,15 @@ def _show_edp(self, message, env, dep, play):
                             refs.append(
                                 "{}_version={}".format(key, value.split()[1]))
 
-                def instance_name(instance):
-                    """
-                    Return name of the instance.
-                    """
-                    return instance.name
-                elbs = map(instance_name,
-                           self._instance_elbs(instance.id, dep))
+                elb_list = []
+                for elb in elbs:
+                    lb_instance_ids = [inst.id for inst in elb.instances]
+                    if instance.id in lb_instance_ids:
```

Comment:
I assume there's some clever python generator thing we could do here, but elb.instances should be already fetched from the API.  Traces in the ticket already show a huge API win.